reference/api/stats.md

```diff
@@ -78,3 +94,5 @@ Get stats of all indexes.
   }
 }
 ```
+
+In the above code sample, `fieldDistribution` shows every field in the `movies` and `rangemovies` indexes along with the total number of documents in the database that contain that field.
```

Comment:
> along with the total number of documents in the database that contain that field.

This is not accurate; if it was, the example would show 19654 + 19654 = 39308 for `overview`, `id`, and `title`.